dfe-analytical-services/dfeR
# Feature/publication themes

**Background**

To resolve #29. 

**Notes**

1. Rather than using use_style I've found that the best way is to make custom document output types. I've made functions for national_statistic and official_statistic that can be referred to as output_format in Rmarkdowns via dfeR::national_statistic and dfeR::official_statistic. You can also get a template with all the markdown examples via File > New file > R Markdown > From Template.
2. I've removed yams as a dependency as its been messing up installs. it itself is a dependency of rmarkdown so that should cover us.
3. I've left out the other template for now that we use as want to have more of a think about it with the cover pages and whatnot.

**QA**

1. Check the code looks ok. Its just using the word_document function so should be fine
2. Check that the word doc template outputs look ok. I slightly edited them after you changed them as they were still a bit out of sync.
3. Check documentation is clear
4. Anything else that you think is needed to prepare for use. Hopefully these will be useful to data group.
